packages/main/src/Card.js

```diff
@@ -116,8 +143,9 @@ class Card extends UI5Element {
 		return !!this.header.length;
 	}
 
-	get _ariaCardRoleDescription() {
-		return this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD);
+	get _getAriaLabelledby() {
```

Comment:
better to name this method _getAriaLabel as it concerns the `aria-label` attribute